index.json

```diff
@@ -20593,16 +20593,13 @@
   },
   {
     "name": "repository_dispatch",
-    "description": "This event occurs when a GitHub App sends a `POST` request to the \"[Create a repository dispatch event](https://docs.github.com/en/v3/repos/#create-a-repository-dispatch-event)\" endpoint.\n\nGitHub Apps must have the `contents` permission to receive this webhook.",
+    "description": "This event occurs when a GitHub App sends a `POST` request to the \"[Create a repository dispatch event](https://docs.github.com/en/v3/repos/#create-a-repository-dispatch-event)\" endpoint.",
```

Comment:
the description is directly parsed from the website, this change will be reverted with next next automated update

Reply:
I haven't edited `index.json` manually – I ran `bin/octokit-webhooks.js update` and that's what was generated.

Reply:
As far as I can see, they've recently moved this note to a separate section: https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#availability-35.